bin/agent-data-plane/src/internal/remote_agent.rs

```diff
@@ -204,9 +205,25 @@ impl RemoteAgent for RemoteAgentImpl {
     async fn get_flare_files(
         &self, _request: tonic::Request<GetFlareFilesRequest>,
     ) -> Result<tonic::Response<GetFlareFilesResponse>, tonic::Status> {
-        let response = GetFlareFilesResponse {
-            files: HashMap::default(),
-        };
+        let mut files = HashMap::new();
+
+        let log_file_path = PathBuf::from(
+            std::env::var("DD_ADP_LOG_FILE").unwrap_or(saluki_app::logging::DEFAULT_ADP_LOG_FILE.to_string()),
+        );
```

Comment:
We should be pulling this from the configuration itself, not just an environment variable.